index.markdown

```diff
@@ -1494,7 +1495,8 @@ The main places are really only the Space Needle and Pike Place Market. More wil
 
 <section id="Area-Boston-MA" />
 ### Boston, MA
-#### Food
+
+#### Food ####
```

Comment:
This fixes the issue found [here](https://stephenholiday.com/Unofficial-Waterloo-USA-Intern-Guide/#Area-Boston-MA)